src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/Rfc2898DeriveBytes.cs

```diff
@@ -11,6 +11,9 @@ namespace System.Security.Cryptography
 {
     public class Rfc2898DeriveBytes : DeriveBytes
     {
+        private static readonly UTF8Encoding s_utf8NoBom =
+            new UTF8Encoding(encoderShouldEmitUTF8Identifier: false);
+        
```

Comment:
This shouldn't be needed. The call sites are just using GetBytes, which isn't affected by the BOM, and thus the call sites can just use Encoding.UTF8.GetBytes instead.